recipes/opencv/4.x/conanfile.py

```diff
@@ -80,6 +82,7 @@ def config_options(self):
             del self.options.fPIC
         if self.settings.os != "Linux":
             del self.options.with_gtk
+            del self.options.with_v4l
```

Comment:
I would add an warning or info message, because v4linux will require a system library. Not all distros have v4l installed by default.